src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -181,6 +181,29 @@ function _parseBoxAttributes(element) {
   _parseAttributeWithoutOverride(element, 'icon', true, '_icon');
 }
 
+/*
+ * Dropdowns
+ */
+
+function _parseDropdownAttributes(element) {
+  const el = element;
+  const slotChildren = el.children && el.children.filter(child => _.has(child.attribs, 'slot'));
+  const hasHeaderSlot = slotChildren && slotChildren.some(child => child.attribs.slot === 'header');
+
+  // If header slot is present, the header attribute has no effect, and we can simply remove it.
+  if (hasHeaderSlot) {
```

Comment:
Perhaps we could delete `el.attribs.text` here, try to add deprecation notes for `text` where appropriate as well